README.md

```diff
@@ -517,6 +520,37 @@ IMAGE_PROCESS_CLASS_PREFIX = "custom-prefix-"
 IMAGE_PROCESS_ADD_CLASS = False
 ```
 
+#### Converting Image Paths to URLs in Metadata
+
+If you want *Image Process* to process images in the metadata
+of your content (for example, in the `og_image` field used by the `seo` and  `pelican-open_graph` plugins),
+you can set the `IMAGE_PROCESS_METADATA` setting to a dictionary mapping
+metadata field names to transformation names. For example:
+
+```python
+IMAGE_PROCESS_METADATA = {
+    "og_image": "og-image-transform",
+}
+```
```

Comment:
Could you provide an example `IMAGE_PROCESS` here, showing the listed transformation (maybe scale to the proper dimensions??)?